apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/CourseNumberSearchBar.tsx

```diff
@@ -72,20 +96,28 @@ class CourseNumberSearchBar extends PureComponent<Record<string, never>, CourseN
                     value={this.state.courseNumber}
                     onChange={this.handleNumbersChange}
                     helperText="ex. 6B, 17, 30-40"
+                    style={{ marginRight: 15 }}
                 />
 
-                <FormControlLabel
-                    control={
-                        <Switch
-                            onChange={this.handleUpperDivChange}
-                            value="100-199"
-                            color="primary"
-                            checked={this.state.courseNumber === '100-199'}
-                        />
-                    }
-                    labelPlacement="top"
-                    label="Upper Div Only"
-                />
+                <FormControl style={{ minWidth: 100 }}>
+                    <InputLabel>Course Level</InputLabel>
+                    <Select
+                        value={this.state.courseLevel}
+                        renderValue={(selected) =>
+                            courseLevelList.find((item) => item.level === selected)?.label as string
```

Comment:
super-duper-ultra-nit: I don't like the parenthesis around the sole parameter of the arrow function when it's nested immediately inside another function because it makes it _slightly_ harder to see where the function ends. Feel free to ignore if you disagree.